packages/@aws-cdk/aws-elasticloadbalancing/lib/load-balancer.ts

```diff
@@ -49,6 +49,16 @@ export interface LoadBalancerProps {
    * @default - None.
    */
   readonly healthCheck?: HealthCheck;
+
+  /**
+   * Whether cross zone load balancing is enabled
+   *
+   * This controls whether the load balancer evenly distributes requests
+   * across each availability zone
+   *
+   * @default - false.
+   */
+  readonly crossZone?: boolean;
```

Comment:
Does `true` make more sense as a default value here? I would feel better with that as a default value, and it seems the ELB team agrees:

According to: https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/enable-disable-crosszone-lb.html

> With the AWS Management Console, the option to enable cross-zone load balancing is selected by default. 

Given that we kind of see the CDK as the "AWS console but without clicking", it makes sense to me that the recommended defaults would be the same between console and CDK.

Reply:
@rix0rrr good suggestion! I was just merely following the CloudFormation documentation.

I've updated my PR based on your recommendation. Let me know what you think.